src/tokens/utils.ts

```diff
@@ -25,6 +25,19 @@ export function sortByUsdBalance(token1: TokenBalance, token2: TokenBalance) {
   return token2UsdBalance.comparedTo(token1UsdBalance)
 }
 
+export function sortByUsdBalanceThenByAlphabetical(token1: TokenBalance, token2: TokenBalance) {
```

Comment:
If this is used only in the `SwapScreen` I'd personally rather have this function live with that code and renamed to be, like, `tokenCompare` to match existing naming conventions (e.g., `.comparedTo` or `localeCompare`, etc).

Reply:
Sure, I can move it there